src/ui/popups/info.html

```diff
@@ -30,6 +30,10 @@
 			<div class="albumArtist">
 				<a id="albumArtist" href="#" target="_blank"></a>
 			</div>
+			<div class="user-play-count">
```

Comment:
Based on what @alexesprit was saying we could use something like https://fontawesome.com/icons/lastfm to indicate this is from last.fm